packages/account-api/src/api/group.ts

```diff
@@ -35,6 +35,24 @@ export enum AccountGroupType {
  */
 export type AccountGroupId = `${AccountWalletId}/${string}`;
 
+/**
+ * Regex to validate a valid account group ID.
+ */
+export const ACCOUNT_GROUP_ID_REGEX =
```

Comment:
Could having a `ACCOUNT_GROUP_ID_REGEX` and `MULTICHAIN_ACCOUNT_GROUP_ID_REGEX` cause confusing?

Reply:
They do match their respective types `AccountGroupId` which is the "common definition" for account groups. And `MultichainAccountGroupId` (which is compatible with `AccountGroupId` OFC) which is a more "specialized" definition for entropy-based (HD) wallets.

We should probably aim to have better doc around those to explain the differences IMO, but we should keep naming "aligned" with each variants.